CHANGELOG.md

```diff
@@ -3,7 +3,25 @@
 This file documents all notable changes in `LogDNA Node.js Code Library`. The release numbering uses [semantic versioning](http://semver.org).
 
 ## Next Release [MAJOR]
```

Comment:
I haven't noticed before but why are we saying it's `[MAJOR]` change? it's not - in terms of `SemVer = [MAJOR].[MINOR].[PATCH]`, this is gonna be `[MINOR]` change

Reply:
@smusali This is not a backward-compatible change. This will require clients to add `shimProperties` option when calling `Logger.createLogger`